fastdup/models/grounding_dino.py

```diff
@@ -0,0 +1,279 @@
+import logging
+import os
+import subprocess
+
+try:
+    from mmengine.config import Config
+    from PIL import Image
+    import torch
+    import groundingdino
+    import groundingdino.datasets.transforms as T
+    from groundingdino.models import build_model
+    from groundingdino.util import get_tokenlizer
+    from groundingdino.util.utils import clean_state_dict, get_phrases_from_posmap
+
+    grounding_dino_transform = T.Compose(
+        [
+            T.RandomResize([800], max_size=1333),
+            T.ToTensor(),
+            T.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225]),
+        ]
+    )
+except ImportError as e:
+    print(e)
```

Comment:
@dnth please use fastdup_sentry_capture_exception to report any exception (vs. print(e))